projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css

```diff
@@ -4,54 +4,75 @@
 .wrapper {
   position: relative;
 }
-.wrapper > .primary {
+
+.wrapper>.primary {
   display: flex;
 }
+
 button.working {
   position: relative;
 }
+
 button[disabled] {
   color: #adc8d1;
   background-color: #7FA9B7;
   cursor: default;
 }
+
 button.plain {
-  background-color: #bbb;
+  background-color: #595959;
```

Comment:
Variables didn't work in this file???

Reply:
That is because it is a css file.

Reply:
You can change the file extension to `.scss` and update the `styleUrls` prop of the button component to use that file. You should then be able to import the variables.